content/blog/2020-09-25-core-devs-call-2020-ecip-1092-1097-last-call/index.en.md

```diff
@@ -10,13 +10,18 @@ tags: ["announcement", "development", "education", "teams", "hardfork", "media"]
 ## ETC Core Devs Call - ECIP-1092 and ECIP-1097 51% Attack Solutions
 
 * When: Friday, September 25, 2020, 4pm UTC, 90 minutes max.
-* Where: Jitsi link will be published ahead of the call.
-* Focus: 51% Attack Solutions
+* Where: [Discord](https://discord.gg/hQs894U)
+* Focus: Mining Algorithm, Last-Call Proposals, and other 51%-Attack Solutions.
 
 ## Agenda
 
+51% Proposals
 * ECIP 1092 PIRLGUARD (Draft)
-* ECIP 1097 Ouroboros BFT Checkpointing (Draft)
```

Comment:
Recommending to leave this line in. IOHK devs / researchers were on the call to discuss and answer questions by participants.

Would also suggest adding "ECIP-1096 ECIP 1096: 51% Attack protection system based on Bitcoin Merged Mining"  as they were also in the call explaining and answering questions about their proposal. https://github.com/ethereumclassic/ECIPs/pull/344

These two along with MESS were the only 51% solutions discussed by respective authors.

Reply:
Resolved. These were added. I believe you're viewing a dated version of this file. My last commit should have added all the notes posted after the meeting.